creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -0,0 +1,149 @@
+const redisClient = require('../redis')
+const { logger: genericLogger } = require('../logging')
+
+const DailyRedisKeyExpirationSec = 90 /* days */ * 24 /* hr */ * 60 /* min */ * 60 /* s */
+
+const Outcomes = Object.freeze({
+  SUCCESS: 'Success',
+  FAILURE: 'Failure'
+})
+
+const SecondarySyncHealthTracker = {
+  async recordSuccess (secondary, wallet, syncType, logContext = {}) {
```

Comment:
why have logContext passed in and create a genericLogger child vs just using genericLogger?

Reply:
you right, fixed